### PR TITLE
Training server will now save ONNX models

### PR DESCRIPTION
**Pull request by rvk007**

### CODE REVIEW

File: train_config/training_server/flash.py [hunk 1]
```diff
@@ -64,9 +69,10 @@ def main():
         train_config['token'],
         train_config['task_type'],
         accuracy,
-        model_path,
+        model_path_pt,
         acc_plot_path,
         metadata,
+        model_path_onnx=model_path_onnx if model_path_onnx else None
```
Comment (shan18) [inline train_config/training_server/flash.py:75]: This will throw an error. Define this variable on top as `None`

**Review by shan18 — CHANGES_REQUESTED** [review]

File: train_config/training_server/image_classification/classification.py [hunk 2]
```diff
@@ -45,8 +48,11 @@ def train_img_classification(config):
     # Train model
     model = fit_model(config, model, train_loader, val_loader, device, checkpoint_path)
 
-    # Save model
-    save_model(os.path.join(checkpoint_path, 'model.pt'), target_model_path)
+    # Save pt model
+    save_model_pt(os.path.join(checkpoint_path, 'model.pt'), target_model_path_pt)
+
+    # Save onnx model
+    save_model_onnx(os.path.join(checkpoint_path, 'model.pt'), target_model_path_onnx)
```
Comment (shan18) [inline train_config/training_server/image_classification/classification.py:55]: Change extension name

**Review by shan18 — CHANGES_REQUESTED** [review]

File: train_config/training_server/image_classification/classification.py (hunk 2 above)
Comment (rvk007) [reply train_config/training_server/image_classification/classification.py:55]: This is not an error. We are converting to onnx model from .pt model.

**Review by shan18 — APPROVED** [review]

File: train_config/training_server/flash.py (hunk 1 above)
Comment (rvk007) [reply train_config/training_server/flash.py]: Resolved in the latest push.

**Pull request merged**